### PR TITLE
[v5] Remove `autoForward`

### PR DESCRIPTION
**Pull request by davidkpiano**
The recommended pattern moving forward would be to explicitly `sendTo(...)` events to actors.

~~There are still some SCXML tests that depend on `autoForward`.~~ XState will no longer support `autoForward` in SCXML.

### CODE REVIEW

File: packages/core/test/invoke.test.ts [hunk 1]
```diff
@@ -232,7 +235,7 @@ describe('invoke', () => {
     service.send({ type: 'FORWARD_DEC' });
   });
 
-  it('should forward events to services if autoForward: true before processing them', (done) => {
+  it('another forwarding pattern', (done) => {
```
Comment (Andarist) [inline packages/core/test/invoke.test.ts:238]: IMHO this test could just be removed, it was testing a specific thing which is no longer relevant and the new test title is meaningless

File: packages/core/test/invoke.test.ts [hunk 2]
```diff
@@ -162,7 +164,7 @@ describe('invoke', () => {
       .start();
   });
 
-  it('should forward events to services if autoForward: true', () => {
+  it('can forward events to services', () => {
```
Comment (Andarist) [inline packages/core/test/invoke.test.ts:167]: This test title is also kinda irrelevant now, we are not even forwarding anything here, just sending an event to a child from within a transition action. Perhaps we should rename this to something like:
```suggestion
  it('child can immediately respond to the parent with multiple events', () => {
```

File: packages/core/test/invoke.test.ts [hunk 3]
```diff
@@ -118,16 +118,18 @@ describe('invoke', () => {
           start: {
             invoke: {
               src: 'child',
-              id: 'someService',
-              autoForward: true
+              id: 'someService'
```
Comment (Andarist) [inline packages/core/test/invoke.test.ts:121]: similarly, this test doesn't quite bring any value, I think - it's almost identical to the one below it

Comment (davidkpiano) [reply packages/core/test/invoke.test.ts:121]: Should we delete it?

Comment (Andarist) [reply packages/core/test/invoke.test.ts:121]: i think so

File: packages/core/test/invoke.test.ts (hunk 1 above)
Comment (Andarist) [reply packages/core/test/invoke.test.ts:238]: 👋

Comment (davidkpiano) [reply packages/core/test/invoke.test.ts:238]: Will delete

**Pull request merged**